### PR TITLE
remove inchi etc from Substance

### PR DESCRIPTION
**Pull request by turbomam**
reference knowledge should be captured on `ChemicalEntity` or substance category enum (?) permissible values

### CODE REVIEW

**Comment by turbomam** [comment]
some reformatting "sneaked in"

File: src/schema/core.yaml [hunk 1]
```diff
@@ -293,10 +293,11 @@ classes:
       - volume
       - mass
 
-      - chemical_formula
-      - inchi
-      - inchi_key
-      - smiles
+  ## MAM 2024-03-26: I meant to delete these ina  previous PR
```
Comment (turbomam) [inline src/schema/core.yaml:296]: commented out these slots

**Comment by turbomam** [comment]
Thanks @anastasiyaprymolenna that's great feedback.

Is it safe to say that this PR can be merged in advance of your tasks 2 and 3?

I agree that this PR has not left `Solution` in a good state. @sierra-moxon and I talked about it some more since we last meet. I will work on it some more now, but maybe the three of us could meet today or early next week?

**Comment by anastasiyaprymolenna** [comment]
@turbomam yeah I think this can be merged. I will create a subsequent PR to fix `Solution` class. I feel like this PR has enough file changes, any more commits would be too much. The other tasks can be their own issues to do once this is merged.  I will PM to set up a time to meet.

**Review by anastasiyaprymolenna — APPROVED** [review]

**Comment by turbomam** [comment]
thank you. I'm looking forward to our next meeting.

**Pull request merged**